OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -290,6 +291,17 @@ public interface IProvidesAssetBrowserColorPickerPalettes
 		IEnumerable<string> ColorPickerPaletteNames { get; }
 	}
 
+	public interface IColorPickerManagerInfo : ITraitInfoInterface
+	{
+		Color ColorPickerPaletteColor { get; set; }
+		(float hMin, float hMax, float sMin, float sMax, float vMin, float vMax) GetHSVRange { get; }
```

Comment:
IMO this is a really ugly method signature. Can we please split this into `GetHueRange`, `GetSaturationRange`, `GetValueRange`?